CodeWars Day 11/Solution11.js

```diff
@@ -0,0 +1,12 @@
+function cubeOdd(arr) {
+  for(var i=0; i < arr.length; i++){
+    if (isNaN(arr[i])) {
+      return undefined;
+    }
+  }
+  
+  return arr
+  .map(a => Math.pow(a,3))
+  .filter(a=> Math.pow(a%2,2)===1)
+  .reduce((acc, val) => val+acc, 0);
+}
```

Comment:
So instead of creating an extra loop for checking if undefined I would just do your map, filter, reduce and check if the result of that is NaN.

https://repl.it/repls/CandidDiligentDrawings